src/main/java/com/zotov/edu/passportofficerestservice/controller/dto/request/PersonRequest.java

```diff
@@ -0,0 +1,31 @@
+package com.zotov.edu.passportofficerestservice.controller.dto.request;
+
+import com.zotov.edu.passportofficerestservice.controller.validator.Country;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.time.LocalDate;
+
+@AllArgsConstructor
+@NonNull
+@Getter
+public final class PersonRequest {
+
+    @NotNull
+    @NotEmpty
+    private final String name;
+
+    @NotNull
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    private final LocalDate birthday;
+
+    @NotNull
+    @Pattern(regexp = "[a-zA-Z]{2}")
```

Comment:
This seems redundant - when `ABC` is passed, both `@Country` and `@Pattern` will be triggered

Reply:
The annotation has been removed.